.gitmessage

```diff
@@ -17,3 +17,4 @@ Any links to any relevant tickets, articles or other resources?
 # Co-authored-by: Nathan Acuff <nathan@expectedbehavior.com>
 # Co-authored-by: Matthew Gordon <matt@expectedbehavior.com>
 # Co-authored-by: Jonathon Fruchte <jonathon@expectedbehavior.com>
+# Co-authored-by: David Ronk <davidronk@gmail.com>
```

Comment:
If you go to GitHub email settings (https://github.com/settings/emails), you can add your EB email, while still keeping your gmail email as your GitHub public email.

Reply:
... also looking at you, @tonydewan

Reply:
David already had his EB email address in that list, but the co-authorship line didn't have the desired impact when using it. You can compare [this commit](https://github.com/expectedbehavior/instrumental_web/commit/ec567d2dbcabc7be70f407166c6df6637748e05e) to [this one](https://github.com/expectedbehavior/instrumental_web/commit/7322dfbd0329848f6f2441347d49833a60837647) that was force-pushed over it. That led us to believe the co-authorship reference must be the primary email address of the GitHub account.

Reply:
weird.  okay.  it's fine then :)

Reply:
To add to the conversation Jason just made [this commit](https://github.com/expectedbehavior/instrumental_web/pull/1578/commits/02bc6f492eb9972ee07a0bcb4c4f610277648329).  He co-authored with my EB account and it worked correctly.
The only difference between Tony's commit and Jason's that I can think of is that I made my EB email a "backup" in settings.
Tony and I may do some tests to verify.

Reply:
David and I just did some thorough testing, and it seems like either we were wrong before or GitHub has fixed something in the last 2 days.

This seems to be true: as long as you have your EB email address as a verified email address on your account, the co-authorship line will work with that email address. It does not need to be your primary nor does it need to be the backup.